### PR TITLE
Add a blog post for Tenzir v4.4

### PR DESCRIPTION
**Pull request by dominiklohmann**
This adds the blog post for the release planned for today. It's a bit fluffy since most of the content already was split out into two blog posts, but I think that's fine.

Filing this before the actual release PR so the blog post is actually live at the time of the release.

### CODE REVIEW

**Review by mavam — APPROVED** [review]
Pushed a wordsmithing commit. GTG.

File: web/blog/tenzir-v4.4/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+title: Tenzir v4.4
+authors: [dominiklohmann]
+date: 2023-11-02
+tags: [release, operators, velociraptor, yara]
```
Comment (mavam) [inline web/blog/tenzir-v4.4/index.md:5]:
```suggestion
tags: [release, operator, velociraptor, yara]
```

File: web/blog/tenzir-v4.4/index.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+title: Tenzir v4.4
+authors: [dominiklohmann]
+date: 2023-11-02
```
Comment (dominiklohmann) [inline web/blog/tenzir-v4.4/index.md:4]:
```suggestion
date: 2023-11-03
```

We pushed the release to tomorrow.

File: web/blog/tenzir-v4.4/index.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+Velociraptor comes with its own language for interacting with it
+programmatically: VQL. The `velocirator` operator makes it possible to submit
+VQL queries to a Velociraptor server, as well as subscribe to artificats
+artifacts in hunt flows over a large fleet of assets, making endpoint telemetry
+collection and processing a breeze.
+
+:::info
+Read our blog post on how we built this integration and how you can utilize it:
+[Integrating Velociraptor into Tenzir
+Pipelines](blog/integrating-velociraptor-into-tenzir-pipelines)
```
Comment (dominiklohmann) [inline web/blog/tenzir-v4.4/index.md:53]:
```suggestion
Pipelines](/blog/integrating-velociraptor-into-tenzir-pipelines)
```

File: web/blog/tenzir-v4.4/index.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+```
+load s3 bucket/file.exe
+| yara path/to/rules/
+```
+
+:::info
+We've written a blog post on the YARA operator that shows just how it works and
+explains in-depth how you can use it: [Matching YARA Rules in Byte
+Pipelines](blog/matching-yara-rules-in-byte-pipelines)
```
Comment (dominiklohmann) [inline web/blog/tenzir-v4.4/index.md:37]:
```suggestion
Pipelines](/blog/matching-yara-rules-in-byte-pipelines)
```

File: web/blog/tenzir-v4.4/index.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  granular control of the auto-restart and auto-delete behavior, the _Stopped_
+  state for pipeline has now been divided into _Stopped_, _Completed_, and
+  _Failed_. The states reflect whether a pipeline was manually stopped, ended
+  naturally, or encountered an error, respectively.
+
+- **Label Support for Pipelines:** You can now visually group related pipelines
+  using the new labels feature. This helps you in organizing your pipelines
+  better for improved visibility and accessibility.
+
+We provide a full list of changes [in our changelog](changelog#v440).
```
Comment (dominiklohmann) [inline web/blog/tenzir-v4.4/index.md:79]:
```suggestion
We provide a full list of changes [in our changelog](/changelog#v440).
```

File: web/blog/tenzir-v4.4/index.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+tags: [release, operators, velociraptor, yara]
+---
+
+[Tenzir v4.4](https://github.com/tenzir/tenzir/releases/tag/v4.4.0) is out!
+We've focused this release on integrations with two pillars of the digital
+forensics and incident response (DFIR) ecosystem: [YARA][yara] and
+[Velociraptor][velociraptor].
+
+[yara]: https://yara.readthedocs.io
+[velocirator]: https://docs.velociraptor.app
```
Comment (dominiklohmann) [inline web/blog/tenzir-v4.4/index.md:14]:
```suggestion
[velocirator]: https://docs.velociraptor.app

```

File: web/blog/tenzir-v4.4/index.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
+---
+title: Tenzir v4.4
+authors: [dominiklohmann]
+date: 2023-11-03
```
Comment (dominiklohmann) [inline web/blog/tenzir-v4.4/index.md:4]:
```suggestion
date: 2023-11-06
```

We moved this to Monday just now, there was little time today to cut the release.

**Pull request merged**